README.md

```diff
@@ -61,6 +61,26 @@ Please note that :
  * `SEO_ENHANCER_OPEN_GRAPH` setting must be enabled in order to use
   `SEO_ENHANCER_TWITTER_CARDS` (see [why](#twitter-cards))
 
+When Open Graph and Twitter Cards features are enabled, the SEO Enhancer will try to add
```

Comment:
Details about how the feature works should be included in [this section](https://github.com/pelican-plugins/seo/blob/main/README.md#twitter-cards) (or in a new section just after the twitter cards), but not here as it's the global explanation of the plugin.